common-test/src/main/java/io/strimzi/test/CmData.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation for test classes or methods run via {@code @RunWith(StrimziRunner.class)}
+ * which causes that runner to update config map before tests
+ */
+@Target({ElementType.METHOD, ElementType.TYPE})
```

Comment:
This annotation is only used by `ConnectCluster.config()` and `KafkaCluster.config()`, it's not intended to be added to methods or classes directly itself, so I think you need `@Target({})`.